lang/contexts.json

```diff
@@ -1,6 +1,7 @@
 {
 	"Paragraph": "Drop-down option label for the paragraph format.",
-	"Heading": "Tooltip and default label for the heading drop-down.",
+	"Heading": "Tooltip for the heading drop-down.",
```

Comment:
Do we need the tooltip at all? If only icon was used it would make a sense, but here we're duplicating the label. And it can give you this too:

![image](https://cloud.githubusercontent.com/assets/156149/24403613/f42da1e6-13bd-11e7-866b-a257f57ec73b.png)

I'm for no label. Why was the label introduced?